config/spamd/spamd_db.php

```diff
@@ -122,35 +119,38 @@
 }
 
 /* spam trap e-mail address */
-if($_GET['spamtrapemail'] <> "") {
```

Comment:
@PiBa-NL You've been fixing "add spamtrap" in 7aac4188282948ac508a1fa3b625dbf514e9d0e4  - I don't get this. Why was it being done twice?